tests/zfs-tests/tests/functional/mv_files/mv_files.cfg

```diff
@@ -44,3 +44,7 @@ export NEWDIR_ACROSS_FS=$TESTDIR_TGT/newdir
 export MVNUMFILES=2000 # <number of files to start>
 export MVNUMINCR=1000  # <number of files to be increased to>
 export GANGPIDS=50 	# <number of limit for parallel background running process>
+
+# controls the "random_creation" test
+export RC_PASS1=$((2**16)) # <number of files to create in the first directory>
+export RC_PASS2=$((2**13)) # <process this many files into the second directory
```

Comment:
Can you write out the full number for these?:
```
- export RC_PASS1=$((2**16)) # <number of files to create in the first directory>
- export RC_PASS2=$((2**13)) # <process this many files into the second director
+ export RC_PASS1=65536 # <number of files to create in the first directory>
+ export RC_PASS2=8192  # <process this many files into the second director
```
That would match the convention for the rest of the entries in this file.